courses-app/src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { Button, Card } from 'antd';
+import './courseCard.less'
+export default function CourseCard() {
+  return (
+    <div>
+      <Card style={{ width: '100%', border: '1px solid green', marginBottom: '20rem', marginTop: '20rem' }}>
+        <div>
+          <p className='card-title'>Java</p>
+          <p>xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx</p>

```

Comment:
Component should be designed to be able to used in any other places, so the reusable parts should not be hard coded, please try to use props to pass data